app/src/main/java/org/simple/clinic/storage/monitoring/SQLiteDatabaseSqlDelegate.kt

```diff
@@ -0,0 +1,66 @@
+package org.simple.clinic.storage.monitoring
+
+import android.content.ContentValues
+import android.database.Cursor
+import android.os.CancellationSignal
+import androidx.sqlite.db.SupportSQLiteDatabase
+import androidx.sqlite.db.SupportSQLiteQuery
+
+interface SQLiteDatabaseSqlDelegate {
```

Comment:
The methods here are mostly copied from [SupportSQLiteDatabase](https://developer.android.com/reference/androidx/sqlite/db/SupportSQLiteDatabase).